dwave_networkx/drawing/qubit_layout.py

```diff
@@ -23,13 +23,16 @@
 from networkx import draw
 
 from dwave_networkx.drawing.distinguishable_colors import distinguishable_color_map
+from itertools import repeat, chain
+
+from numbers import Number
 
 __all__ = ['draw_qubit_graph']
```

Comment:
What about funcs ``draw_embedding``, ``draw_yield``, ``normalize_size_and_aspect``, ``draw_lineplot``? Shouldn't these be added? And added to the appropriate RST file?